test_project/tests.py

```diff
@@ -69,3 +74,35 @@ def get_value_label(value):
     assert get_value() == '3'
 
     session_browser.find_by_css('input[type=submit]').click()
+
+
+def test_get_parameter(session_browser, live_server):
```

Comment:
This looks like it doesn't need the browser dependencies so if I understand pytest dependency injection correctly you can simply do away with the both arguments.

Also so you think it's possible to reduce the number of test form classes a bit? I'd prefer to have as few of them as possible because otherwise when reading a unit test you always have to read and understand this huge preamble before being able to understand the meat of the test (i.e. the actual test).